features/environment.py

```diff
@@ -0,0 +1,18 @@
+from behave import *
+import ModelController
+
+def before_all(context):
+    context.MEPath = "/home/seanjkeene/PycharmProjects/doe-egot-me/"
+
+    print("First run...")
+    try:
```

Comment:
Note that, in ModelController.py, the end_program flag invokes the python quit() method. Behind the scenes, this method actually calls a SystemExit exception. This means that if the MC is run as a module within the behave layer, after the first simulation run, the SystemExit exception will occur, breaking out of the MC script but ALSO killing the behave testing. Placing it in an exception handler allows us to run the MC multiple times in a row and then perform tests without the exception killing the whole process.